lib/OpenQA/CLI/api.pm

```diff
@@ -83,6 +87,7 @@ sub command {
     openqa-cli api --osd jobs groupid=135 distri=caasp version=3.0 latest=1
 
     # List the latest jobs matching the search criteria
+    # supported search criteria: $search_criteria
```

Comment:
This is a comment... with a perl variable?

Reply:
Is this expanded via asciidoc or some such?

Reply:
It's not really a comment, it's a POD section where someone used the custom formatting with just a "#" in front of the command examples. The variable is replaced in https://github.com/os-autoinst/openQA/pull/4402/files#diff-4852ca9e62b5150bacb99e7492d2146fdd3af1cbcd4498087dcc9410260e2f8cR15